File: RandomLearning/RandomLearning/ViewController.m
//
//  ViewController.m
//  RandomLearning
//
//  Created by Christopher Wainwright Aaron on 6/2/15.
//  Copyright (c) 2015 Christopher Wainwright Aaron. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    
    [self circleMake];
    


}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

-(void)circleMake {
    
    CGContextRef context = UIGraphicsGetCurrentContext();
    
    CGRect circleViewRect = CGRectMake(50, 100, 100, 100);
    
    UIView * circle = [[UIView alloc] initWithFrame:circleViewRect];
    
    circle.backgroundColor = [UIColor blueColor];
    
    circle.layer.cornerRadius = 50;
    
    [self.view setNeedsDisplay];

}



@end

File: CalculatorFunctions/CalculatorFunctions/ViewController.swift
//
//  ViewController.swift
//  CalculatorFunctions
//
//  Created by Christopher Wainwright Aaron on 5/5/15.
//  Copyright (c) 2015 Chris Aaron. All rights reserved.
//

import UIKit

class ViewController: UIViewController {
    
    var firstNumber = ""
    var secondNumber = ""
    var currentOperation = ""

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }

    @IBAction func pressedButton1(sender: UIButton) {
        
        let number = sender.currentTitle!
        
        if currentOperation.isEmpty {
            
            firstNumber += number
            
        } else {
            
            secondNumber += number
        }
        
        
     
    }
    
    @IBAction func operatorPressed(sender: UIButton)
    {
        
        let operation = sender.currentTitle!
        
        if operation != "=" {
            
            currentOperation = operation
            
        } else {
            
            switch currentOperation {
                
            case "+":
                
                let value = firstNumber.toInt()! + secondNumber.toInt()!
                println("\(firstNumber) \(currentOperation) \(secondNumber) = \(value)")
                
            case "-":
                println("do subtraction")
            default :
                println("blah")
            }
        
            
            firstNumber = ""
            secondNumber = ""
            currentOperation = ""
            
        }
        
    }
    
    
    
}

